python/shotgun_model/data_handler_nav.py

```diff
@@ -146,10 +152,40 @@ class and the data payload from Shotgun should be provided via the
         num_modifications = 0
 
         new_uids = set()
+
+        # a list of sg data dicts to display items for
+        child_data = []
+
+        if item_path == self.__root_path:
+            self._log_debug("This is the root of the tree.")
+            parent_uid = None
+
+            if self.__include_root:
```

Comment:
I'd suggest that the unselected, default view is the best view for the project context because it shows you the child folders on the right. If it selected the special item, it would only show top-level publishes (not the subfolders). It feels right to me as is. Making the change you suggest doesn't seem to change the behavior.